backend-api/src/functions/tests/pact/asyncCredential/asyncCredentialStateConfiguration.ts

```diff
@@ -0,0 +1,78 @@
+import { Dependencies } from "../../../asyncCredential/handlerDependencies";
+import { registeredLogs } from "../../../asyncCredential/registeredLogs";
+import { MockEventWriterSuccess } from "../../../services/events/tests/mocks";
+import { Logger } from "../../../services/logging/logger";
+import { MockLoggingAdapter } from "../../../services/logging/tests/mockLogger";
+import {
+  MockClientRegistryServiceGetPartialClientSuccessResultIPV,
+  MockSessionServiceCreateSessionSuccessResult,
+  MockTokenServiceInvalidSignatureIPV,
+  MockTokenServiceSuccessIPV,
+} from "../../../testUtils/asyncCredentialMocks";
+
+const env = {
+  SIGNING_KEY_ID: "mockKid",
+  ISSUER: "mockIssuer",
+  SESSION_TABLE_NAME: "mockTableName",
+  SESSION_TABLE_SUBJECT_IDENTIFIER_INDEX_NAME: "mockIndexName",
+  SESSION_TTL_IN_MILLISECONDS: "12345",
+  SQS_QUEUE: "mockSqsQueue",
+  CLIENT_REGISTRY_PARAMETER_NAME: "mockParmaterName",
+};
+
+export class AsyncCredentialStateConfiguration {
```

Comment:
Let's chat about this - it doesn't have much to do with state and I'm not convinced this adds the right level of value

Reply:
Renamed to remove 'state' for now

Reply:
Note: I believe state handlers are needed to setup the Lambdas to suit each interaction. This class is needed to ensure the setup of the Lambdas can be changed using state handlers in the pact.test.ts file.